solr/contrib/ltr/README.md

```diff
@@ -61,7 +61,7 @@ the techproducts example please follow these steps.
        http://localhost:8983/solr/techproducts/schema/model-store
      * Perform a reranking query using the model, and retrieve the features
 
-       http://localhost:8983/solr/techproducts/query?indent=on&q=test&wt=json&rq={!ltr%20model=svm%20reRankDocs=25%20efi.user_query=%27test%27}&fl=[features],price,score,name
+       http://localhost:8983/solr/techproducts/query?indent=on&q=test&wt=json&rq={!ltr%20model=linear%20reRankDocs=25%20efi.user_query=%27test%27}&fl=[features],price,score,name
```

Comment:
Change lucene-ltr-7.0.0-SNAPSHOT.jar to solr-ltr-7.0.0-SNAPSHOT.jar 10 lines above, this should have been updated with a change that happened last week